k8s/namespaces/ia/ia-case-api/ia-case-api.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-api
+spec:
+  releaseName: ia-case-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/ia-case-api
+  values:
+    java:
+      replicas: 2
```

is there a reason auto scaling isn't enabled? recommended but not required and could be a separate PR

```suggestion
      replicas: 2
      autoscaling:
        enabled: true
        maxReplicas: 5
        targetCPUUtilizationPercentage: 80
```

---

we don't have it in other apps too, let me create separate PR with autoscaling property